### ISSUE TITLE
Missing columns in functions()

### ISSUE THREAD

**Issue by lmangani** [quackscience/duckdb-extension-clickhouse-native#3]
The `functions` table offers a broad variety of types and columns. 

The current parser fails at identifying certain column names and types

### clickhouse native table
```sql
:) SELECT * FROM system.functions LIMIT 1;
   ┌─name───────────────────────────────┬─is_aggregate─┬─case_insensitive─┬─alias_to─┬─create_query─┬─origin─┬─description─┬─syntax─┬─arguments─┬─returned_value─┬─examples─┬─categories─┐
1. │ h3GetIndexesFromUnidirectionalEdge │            0 │                0 │          │              │ System │             │        │           │                │          │            │
   └────────────────────────────────────┴──────────────┴──────────────────┴──────────┴──────────────┴────────┴─────────────┴────────┴───────────┴────────────────┴──────────┴────────────┘



```

### data
Generate a native file with `clickhouse` or `clickhouse-local`
```sql
SELECT *
FROM system.functions
INTO OUTFILE '/tmp/manyfunctions.clickhouse'
FORMAT Native
```

Parse the file using the DuckDB clickhouse native extension:

```sql
D SELECT * FROM clickhouse_native('/tmp/manyfunctions.clickhouse') LIMIT 1;
┌────────────────────────────────────┬──────────────┬──────────────────┬──────────┬──────────────┬────────┬─────────────┬────────────────┬────────────────┬────────────────┬────────────────┬────────────────┐
│                name                │ is_aggregate │ case_insensitive │ alias_to │ create_query │ origin │ description │       C7       │       C8       │       C9       │      C10       │      C11       │
│              varchar               │    int32     │      int32       │ varchar  │   varchar    │ int32  │   varchar   │    varchar     │    varchar     │    varchar     │    varchar     │    varchar     │
├────────────────────────────────────┼──────────────┼──────────────────┼──────────┼──────────────┼────────┼─────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┤
│ h3GetIndexesFromUnidirectionalEdge │            0 │                0 │          │              │ System │             │ <unsupported:> │ <unsupported:> │ <unsupported:> │ <unsupported:> │ <unsupported:> │
└────────────────────────────────────┴──────────────┴──────────────────┴──────────┴──────────────┴────────┴─────────────┴────────────────┴────────────────┴────────────────┴────────────────┴────────────────┘
```

**Comment by adubovikov** [comment]
```
00007f90: 0000 066f 7269 6769 6e46 456e 756d 3828  ...originFEnum8(
00007fa0: 2753 7973 7465 6d27 203d 2030 2c20 2753  'System' = 0, 'S
00007fb0: 514c 5573 6572 4465 6669 6e65 6427 203d  QLUserDefined' =
00007fc0: 2031 2c20 2745 7865 6375 7461 626c 6555   1, 'ExecutableU
00007fd0: 7365 7244 6566 696e 6564 2720 3d20 3229  serDefined' = 2)
``
```

**Comment by lmangani** [comment]
Thanks! Enum implemented:

```sql
D SELECT origin FROM clickhouse_native('/tmp/manyfunctions.clickhouse') LIMIT 1;
┌─────────┐
│ origin  │
│ varchar │
├─────────┤
│ System  │
└─────────┘
```

> ```
> 00007f90: 0000 066f 7269 6769 6e46 456e 756d 3828  ...originFEnum8(
> 00007fa0: 2753 7973 7465 6d27 203d 2030 2c20 2753  'System' = 0, 'S
> 00007fb0: 514c 5573 6572 4465 6669 6e65 6427 203d  QLUserDefined' =
> 00007fc0: 2031 2c20 2745 7865 6375 7461 626c 6555   1, 'ExecutableU
> 00007fd0: 7365 7244 6566 696e 6564 2720 3d20 3229  serDefined' = 2)
> ``
> ```

The issue with the unknown columns persists.....

**Comment by lmangani** [comment]
The missing columns appear to be simple strings

```sql
DESCRIBE TABLE system.functions

Query id: ead77f8e-5f30-41e6-bce2-4fa7d7e6c99d

    ┌─name─────────────┬─type───────────────────────────────────────────────────────────────────┬─default_type─┬─default_expression─┬─comment───────────────────────────────────────────────────────┬─codec_expression─┬─ttl_expression─┐
 1. │ name             │ String                                                                 │              │                    │ The name of the function.                                     │                  │                │
 2. │ is_aggregate     │ UInt8                                                                  │              │                    │ Whether the function is an aggregate function.                │                  │                │
 3. │ case_insensitive │ UInt8                                                                  │              │                    │ Whether the function name can be used case-insensitively.     │                  │                │
 4. │ alias_to         │ String                                                                 │              │                    │ The original function name, if the function name is an alias. │                  │                │
 5. │ create_query     │ String                                                                 │              │                    │ Obsolete.                                                     │                  │                │
 6. │ origin           │ Enum8('System' = 0, 'SQLUserDefined' = 1, 'ExecutableUserDefined' = 2) │              │                    │ Obsolete.                                                     │                  │                │
 7. │ description      │ String                                                                 │              │                    │ A high-level description what the function does.              │                  │                │
 8. │ syntax           │ String                                                                 │              │                    │ Signature of the function.                                    │                  │                │
 9. │ arguments        │ String                                                                 │              │                    │ What arguments does the function take.                        │                  │                │
10. │ returned_value   │ String                                                                 │              │                    │ What does the function return.                                │                  │                │
11. │ examples         │ String                                                                 │              │                    │ Usage example.                                                │                  │                │
12. │ categories       │ String                                                                 │              │                    │ The category of the function.                                 │                  │                │
    └──────────────────┴────────────────────────────────────────────────────────────────────────┴──────────────┴────────────────────┴───────────────────────────────────────────────────────────────┴──────────────────┴────────────────┘


```

**Comment by lmangani** [comment]
Hopefully resolved with 2a2563c9ca4219b608ede32911170b17a8af130d (needs testing)
```sql
D SELECT * FROM clickhouse_native('/tmp/manyfunctions.clickhouse') LIMIT 1;
┌──────────────────────┬──────────────┬──────────────────┬──────────┬──────────────┬─────────┬─────────────┬─────────┬───────────┬────────────────┬──────────┬────────────┐
│         name         │ is_aggregate │ case_insensitive │ alias_to │ create_query │ origin  │ description │ syntax  │ arguments │ returned_value │ examples │ categories │
│       varchar        │    int32     │      int32       │ varchar  │   varchar    │ varchar │   varchar   │ varchar │  varchar  │    varchar     │ varchar  │  varchar   │
├──────────────────────┼──────────────┼──────────────────┼──────────┼──────────────┼─────────┼─────────────┼─────────┼───────────┼────────────────┼──────────┼────────────┤
│ h3GetIndexesFromUn…  │            0 │                0 │          │              │ System  │             │         │           │                │          │            │
└──────────────────────┴──────────────┴──────────────────┴──────────┴──────────────┴─────────┴─────────────┴─────────┴───────────┴────────────────┴──────────┴────────────┘
```

**Comment by lmangani** [comment]
Resolved
